easybuild/easyblocks/a/aocc.py

```diff
@@ -116,6 +117,9 @@ def make_module_extra(self):
         txt += self.module_generator.set_environment('ASAN_SYMBOLIZER_PATH', asan_symbolizer_path)
         # setting the AOCChome path
         txt += self.module_generator.set_environment('AOCChome', self.installdir)
+        # make sure clang/flang picks up GCCcore as GCC toolchain
+        txt += self.module_generator.set_alias('clang', 'clang --gcc-toolchain=$EBROOTGCCCORE')
+        txt += self.module_generator.set_alias('flang', 'flang --gcc-toolchain=$EBROOTGCCCORE')
```

Comment:
@SebastianAchilles Maybe we should install an actual wrapper script rather than just defining an alias, since an alias is not inherited into a subshell?

Reply:
Yes, a wrapper script sounds like a good idea. I implemented the wrapper in https://github.com/easybuilders/easybuild-easyblocks/pull/2538/commits/86aa43b7b7cdeb1fc7ab3ffab44b581798550d9b and https://github.com/easybuilders/easybuild-easyblocks/pull/2538/commits/235484cbee1677ccfdae1c262dbfa66f7c4bb319